src/editor.scss

```diff
@@ -35,16 +35,20 @@ svg.ugb-stackable-icon-gradient {
 }
 
 // Gutenberg v5.9 adds top & bottom margins in the editor, get rid of them.
-.editor-styles-wrapper [data-type^="ugb/"] {
-	h1,
-	h2,
-	h3,
-	h4,
-	h5,
-	h6,
-	p {
-		margin-top: 0;
-		margin-bottom: 0;
+.editor-styles-wrapper {
+	// Removed the default padding in editor wrapper in WP 5.5
+	padding: 0 !important;
+	&[data-type^="ugb/"] {
+		h1,
+		h2,
+		h3,
+		h4,
+		h5,
+		h6,
+		p {
+			margin-top: 0;
+			margin-bottom: 0;
+		}
```

Comment:
I don't think this is the right fix also. I would suggest that you look at how the native blocks are doing it. When you make a native Cover block fullwidth, why doesn't it have any gaps on the sides?